### PR TITLE
[1LP][RFR] CB: display on test

### PR DESCRIPTION
**Pull request by digitronik**
Purpose or Intent
=================
- Automated BZ-1653193
- Added BZ-1659452

{{pytest: cfme/tests/automate/custom_button/test_service_objects.py::test_custom_button_text_display -vv}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by digitronik** [comment]
for `Dropdown` read method need https://github.com/RedHatQE/widgetastic.patternfly/pull/79. but for now good to go... BZ already blocking groups.

File: cfme/tests/automate/custom_button/test_service_objects.py [hunk 1]
```diff
@@ -260,3 +289,37 @@ def test_custom_button_automate(request, appliance, context, submit, objects, bu
...
+        testSteps:
+            Prerequisites:
+                * Appliance with Service
+            Steps:
+                * Create custom button `Group` or `Button` without display option
+                * Check Group/Button text display or not on UI and SSUI.
+    """
+
+    my_service = MyService(appliance, name=service.name)
+    button, group = button_group
```
Comment (mshriver) [inline cfme/tests/automate/custom_button/test_service_objects.py]: These are backwards from what is yielded in the fixture.

File: cfme/tests/automate/custom_button/test_service_objects.py [hunk 2]
```diff
@@ -260,3 +289,37 @@ def test_custom_button_automate(request, appliance, context, submit, objects, bu
                 assert False, "Expected {count} requests not found in automation log".format(
                     count=str(expected_count)
                 )
+
+
+@pytest.mark.meta(blockers=[BZ(1659452, forced_streams=["5.9", "5.10"],unblock=lambda button_group: "group" not in button_group)])
```
Comment (mshriver) [inline cfme/tests/automate/custom_button/test_service_objects.py]: `button_group` is a tuple of objects, I don't think this unblock lambda is effective?

File: cfme/tests/automate/custom_button/test_service_objects.py (hunk 1 above)
Comment (digitronik) [reply cfme/tests/automate/custom_button/test_service_objects.py]: my bad; it was creating fixture conflict and bad use. corrected

File: cfme/tests/automate/custom_button/test_service_objects.py (hunk 2 above)
Comment (digitronik) [reply cfme/tests/automate/custom_button/test_service_objects.py]: yes, In such case this is best option. we are not upgraded `pytest`; with this version, we have to go with this. @mshriver I tested this by running locally. but still, if you have any suggestion please let me know.

File: cfme/tests/automate/custom_button/test_service_objects.py [hunk 3]
```diff
@@ -260,3 +288,46 @@ def test_custom_button_automate(request, appliance, context, submit, objects, bu
...
+                3. Check Group/Button text display or not on UI and SSUI.
+    """
+
+    my_service = MyService(appliance, name=service.name)
+    button, group = serv_button_group
+
+    with appliance.context.use(context):
+        navigate_to = ssui_nav if context is ViaSSUI else ui_nav
+        destinations = (
+            ["Details"] if (BZ(1650066, forced_streams=["5.9", "5.10"]).blocks and context is ViaSSUI) else ["All", "Details"]
```
Comment (jawatts) [inline cfme/tests/automate/custom_button/test_service_objects.py]: Could you double check the formatting here? I think it is right when looking at the raw text.

Comment (digitronik) [reply cfme/tests/automate/custom_button/test_service_objects.py]: @jawatts those are right.

Comment (mshriver) [reply cfme/tests/automate/custom_button/test_service_objects.py]: This appears to be 127 char long?

**Pull request merged**